src/visualize/image_like.jl

```diff
@@ -127,35 +109,114 @@ function play{T}(buffer::Array{T, 2}, video_stream, t)
     return reinterpret(T, buffer, (w,h))
 end
 
-"""
-Takes a 3D image and decides if it is a volume or an animated Image.
-"""
-function _default{T<:Colorant, X}(img::Images.Image{T, 3, X}, s::Style, data::Dict)
-    props = img.properties
-    if haskey(props, "timedim")
-        if haskey(props, "spatialorder")
-            get!(data, :spatialorder, join(props["spatialorder"], ""))
+# If the user is using the new Images, ImageAxes will be loaded
+if isdefined(Images, :ImageAxes)
+    unwrap(img::Images.ImageMeta) = unwrap(data(img))
```

Comment:
Oh yeah, I definitely wouldn't like copying, since it will already get copied when transferred to the GPU ;)
Maybe, I should not convert them at all, but do this inside GLAbstraction when copying to the device.
I have more information there, e.g. for buffers, I could even copy from views, since I can get a pointer directly to the GPU memory ;)